how-to/workspace-platform-starter/docs/how-to-define-apps-fdc3-1-2.md

```diff
@@ -71,8 +71,17 @@ The following field is custom to this platform and is optional:
 - launchPreference - This is an optional and new setting added to v15 of workspace-platform-starter. This lets the directory owner or the app owner specify a preference that should apply to the application when launching.
   - bounds - Bounds can optionally be specified. A view is something that exists in a platform/browser window so this allows a preference for height/width to be specified. A classic window can already specify a height and width in it's manifest but you may want a custom height/width for this app entry (especially if the window is defined as an external manifest as opposed to an inline one).
   - defaultCentered - a particular view/classic window might need to be centered on launch. This allows the app/directory owner to specify that preference.
-  - options - Optional options that are specific to the app/manifest type. Only view/inline-view has supported settings at this time.
-    - view - hostUrl lets you specify that this view should be loaded into a specific Platform API window instead of the default browser window. The ability to have Platform API and Browser windows together in the same platform was added in v15 of @openfin/workspace-platform.
+- options - Optional options that are specific to the app/manifest type. Only view/inline-view has supported settings at this time.
+  - view you specify host specific settings.
+    - host - host settings to be used by the page hosting the view
+      - url - lets you specify that this view should be loaded into a specific Platform API window instead of the default browser window. The ability to have Platform API and Browser windows together in the same platform was added in v15 of @openfin/workspace-platform.
+      - title - the title to set on the window and this title is shown if pages are disabled on a browser window
+      - icon - used to replace the menu icon on a browser window (can be used by a platform window if you use the setting)
+      - hasHeader - specify if you wish to have the view tab in a browser or platform api window
```

Comment:
Rename to `hasHeaders` for consistency, as these are the Host preferences

Reply:
done